lib/codegen/fromcto/odata/odatavisitor.js

```diff
@@ -63,7 +63,7 @@ class ODataVisitor {
         } else if (thing.isClassDeclaration?.()) {
             return this.visitClassDeclaration(thing, parameters);
         } else if (thing.isMapDeclaration?.()) {
-            return;
+            this.visitMapDeclaration(thing, parameters);
```

Comment:
should you be returning this?

Reply:
good spot, should do to keep this consistent with the other branches, but functionally will not break if not returned.